src/test/java/com/puppycrawl/tools/checkstyle/CheckerTest.java

```diff
@@ -100,23 +99,21 @@ public void testDestroy() throws Exception {
         checker.addListener(auditAdapter);
         final DebugFilter filter = new DebugFilter();
         checker.addFilter(filter);
+        final TestBeforeExecutionFileFilter fileFilter = new TestBeforeExecutionFileFilter();
+        checker.addBeforeExecutionFileFilter(fileFilter);
 
         // should remove al listeners and filters
         checker.destroy();
 
-        // Let's try fire some events
-        getFireAuditStartedMethod().invoke(checker);
-        getFireAuditFinished().invoke(checker);
-        checker.fireFileStarted("Some File Name");
-        checker.fireFileFinished("Some File Name");
-
-        final SortedSet<LocalizedMessage> messages = new TreeSet<>();
+        checker.process(Collections.singletonList(new File("Some File Name")));
+        final SortedSet messages = new TreeSet<>();
```

Comment:
`SortedSet` => `SortedSet<LocalizedMessage>`

Please always add template type to classes.
Some IDEs will give warnings/errors on this.